test/test_encryption.py

```diff
@@ -1942,6 +1944,44 @@ def test_via_loading_shared_library(self):
         with self.assertRaises(ServerSelectionTimeoutError):
             no_mongocryptd_client.db.command("ping")
 
+    # https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#20-bypass-creating-mongocryptd-client-when-shared-library-is-loaded
+    @unittest.skipUnless(os.environ.get("TEST_CRYPT_SHARED"), "crypt_shared lib is not installed")
+    def test_client_via_loading_shared_library(self):
+        class TCPHandler(socketserver.BaseRequestHandler):
+            def handle(self):
+                assert not self.request.recv(1024)
+
+        def listen():
+            addr = ("localhost", 27021)
```

Comment:
This port is used when testing sharded clusters. Let's change to 47021 like we did in PYTHON-3492.

Reply:
Done.